Library/Formula/packer-windows-plugins.rb

```diff
@@ -0,0 +1,152 @@
+require 'language/go'
+require 'fileutils'
```

Comment:
Both ` fileutils ` and ` pathname ` are required for you automatically. You don't need to reinclude them.